offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CustomerSelectionView.groovy

```diff
@@ -101,11 +103,15 @@ class CustomerSelectionView extends VerticalLayout implements Resettable{
          */
         addButtonsLayout = new HorizontalLayout()
         this.createCustomerButton = new Button("Create Customer", VaadinIcons.USER)
+        this.updatePerson = new Button("Update Customer", VaadinIcons.USER)
```

Comment:
It's fine to use the same icon, we might think about using a separate icon (e.g. the edit button like for the offer overview)